modules/api/pkg/resource/customresourcedefinition/types/types.go

```diff
@@ -59,31 +59,48 @@ type CustomResourceDefinitionDetail struct {
 	Errors []error `json:"errors"`
 }
 
+type AdditionalPrinterColumn struct {
+	Name     string `json:"name"`
+	Type     string `json:"type,omitempty"`
+	Priority int32  `json:"priority,omitempty"`
+	JSONPath string `json:"jsonPath"`
```

Comment:
Maybe we can add a description field as shown [here](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#additional-printer-columns) that can pop up when you hover over the column name?